packages/node_modules/samples/browser-plugin-meetings/test/wdio/spec/mediatest.js

```diff
@@ -13,20 +14,19 @@ describe('samples', () => {
       before('create test users', () => testUsers.create({count: 2})
         .then((users) => {
           [mccoy, spock] = users;
-          console.log('ACCESS TOKEN ', users);
         }));
 
       before('reload browser', () => {
         browser.refresh();
       });
 
       it('loads the app', () => {
-        browser.url('/');
+        browser.url('/browser-plugin-meetings');
```

Comment:
Did someone revert this?

Reply:
I'm not sure, but because it was pointing to root (the table of contents) the entire test failed.